lib/Doctrine/ODM/PHPCR/Mapping/Annotations/Binary.php

```diff
@@ -21,9 +21,13 @@
 
 use Doctrine\Common\Annotations\Annotation;
 
+@trigger_error('The @Binary annotation is deprecated, use @Field(type="binary") instead.', E_USER_DEPRECATED);
```

Comment:
Symfony changed its error strategy in 2.7.1, which is what I'm using here too. As the error is silenced, it isn't visible when using a normal PHP installation. It'll only be visible when using a custom error handler (like the one provided with the Symfony PHPunit bridge or xDebug).

This means that these errors are now opt-in instead of the previous "opt-out"-ish situation of 2.7-dev. That's why I've added it here, but I'm open to comment this line if you don't think it's the correct moment to trigger errors